src/test/scala/za/co/absa/commons/reflect/ClassWithCyclicAnnotation.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.reflect
+
+import za.co.absa.commons.reflection.CyclicAnnotationExample
```

Comment:
I just realized a minor inconsistency we have in a package name - reflection vs reflect. Could you fix it in the scope of this PR please?

```suggestion
import za.co.absa.commons.reflect.CyclicAnnotationExample
```